app/config/PayPalConfig.scala

```diff
@@ -0,0 +1,25 @@
+package config
+
+import com.typesafe.config.Config
+
+class PayPalConfigProvider(config: Config, defaultStage: Stage) extends TouchpointConfigProvider[PayPalConfig](config, defaultStage) {
```

Comment:
I think it would make sense for this to take a `defaultEnvironment: TouchPointEnvironment` instead of `defaultStage` (same for `TouchpointConfigProvider`)